jib-core/src/main/java/com/google/cloud/tools/jib/configuration/credentials/Credential.java

```diff
@@ -21,7 +21,7 @@
 // TODO: Move to lower-level package - probably at same level as Authorization.
```

Comment:
Curious, is there a reason we haven't done this yet?

Reply:
No idea what this means. This looks like an old comment. `Authorization` is under the `.jib.http` package (which doesn't look wrong), and this class does not fit into that package. I'll remove the comment.